extensions/mssql/src/objectManagement/objectManagementService.ts

```diff
@@ -470,7 +470,18 @@ export class TestObjectManagementService implements IObjectManagementService {
 				owner: 'databaseProperties 1',
 				sizeInMb: 16.00,
 				spaceAvailableInMb: 1.15,
-				status: 'Normal'
+				status: 'Normal',
```

Comment:
Looking above, why is there a default date created and size in mb?

(Totally get this isn't part of this PR)

Reply:
If you are referring about the values, I just added them as a sample values to display on UI!